index.js

```diff
@@ -54,3 +66,30 @@ github.repos.fork({
     console.log('Hey it worked!')
   }
 })
+
+function packagePath (dep) {
+  const cwd = process.cwd().match(/([^\/\\]+$)/)[1]
+  if (!dep || cwd == dep) { return 'package.json' }
```

Comment:
Let's default to `===` instead of  `==` due to JavaScript's psychopathy: http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons